examples/oriented_images.html

```diff
@@ -0,0 +1,160 @@
+            }
+        </style>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <script src="js/GUI/dat.gui/dat.gui.min.js"></script>
+        <script src="js/OrientedImageHelper.js"></script>
+    </head>
+    <body>
+        <div id="viewerDiv">
+            <p id="info">
+                Picture taken by Walter Mittelholzer in 1919, metadata from 
+                <a href="https://smapshot.heig-vd.ch/map/?imageId=2873" target="_blank"> smapshot.heig-vd.ch </a>
+            </p>
+        </div>
+        <script src="js/GUI/GuiTools.js"></script>
+        <script src="../dist/itowns.js"></script>
+        <script src="../dist/debug.js"></script>
+        <script type="text/javascript">
+            // Define initial camera position.
+            // Note that camera initial position will be overwritten by picture position later
+            var dummyPositionOnGlobe = { longitude: 0, latitude: 0, altitude: 10000000 };
+
+            // `viewerDiv` will contain iTowns' rendering area (`<canvas>`)
+            var viewerDiv = document.getElementById('viewerDiv');
+
+            // Instanciate iTowns GlobeView*
+            var globeView = new itowns.GlobeView(viewerDiv, dummyPositionOnGlobe, {
+                sseSubdivisionThreshold: 6,
+                noControls: true,
+            });
+
+            var pictureInfos;
+            var coord;
+            var rotationMatrix;
+            // eslint-disable-next-line no-unused-vars
+            var plane;
+            var camera;
+
+            function addLayerCb(layer) {
+                return globeView.addLayer(layer);
+            }
+
+            // Add one imagery layer to the scene
+            // This layer is defined in a json file but it could be defined as a plain js
+            // object. See Layer* for more info.
+            itowns.Fetcher.json('./layers/JSONLayers/Ortho.json').then(addLayerCb);
+
+            // Add two elevation layers.
+            // These will deform iTowns globe geometry to represent terrain elevation.
+            itowns.Fetcher.json('./layers/JSONLayers/WORLD_DTM.json').then(addLayerCb);
+            itowns.Fetcher.json('./layers/JSONLayers/IGN_MNT_HIGHRES.json').then(addLayerCb);
+
+            pictureInfos = {
+                panoramic: {
+                    latitude: 45.9228208442959,
+                    height: 4680.55588294683,
+                    longitude: 6.83256920100156,
+                    azimuth: 526.3900321920207,
+                    roll: 2.1876227239518276,
+                    tilt: -11.668910605126001,
+                    image: 'http://www.itowns-project.org/itowns-sample-data/images/MontBlanc/MontBlanc.jpg',
+                },
+                camera: {
+                    // size of the picture in pixel
+                    size: [6490, 4408],
+                    // focal length in pixel
+                    focal: 8270,
```

Comment:
Why do you request a focal in pixels? exif metadata usually contains the focal in `mm`. If you need pixels => perform the conversion in your code.

Reply:
In photogrammetry (eg [orientation files in micmac](https://github.com/micmacIGN/Documentation/raw/master/DocMicMac.pdf)), you do not primarily care about the focal length in mm : you want to construct the projection matrix and the simplest route is to get everything expressed in pixels : image size, ppa, focal... So that is the other way around. If you have the focal length in mm, you should use your knowledge of the pixel size or sensor size to convert it into a focal length in pixels. Let us for now defer the estimation of the camera parameters from exif metadata to another PR or delegate this to some photogrammetric preprocessing.

Reply:
You keep refering to photogrammetry but the image for this example isn't a coming from a photogrammetry dataset.
That's why it seems strange to me to rely on photogrammetric conventions.